app/lib/account/models.dart

```diff
@@ -199,7 +199,13 @@ class UserSessionData {
     }
     // Double the layout size, for better quality on higher dpi displays.
     final imageSize = layoutSize * 2;
-    return '$imageUrl=s$imageSize';
+
+    // Strip existing options from the imageUrl if there is any
+    var u = imageUrl;
+    if (RegExp(r'=(?:[swh]\d+)|[cp](?:-(?:[swh]\d+)|[cp])*$').hasMatch(u)) {
```

Comment:
Where is this regexp coming from?

Reply:
I wrote it to match the format in https://developers.google.com/people/image-sizing

Reply:
Please note the url in a comment.

Reply:
the URL is higher up, but yes, moving it out I'll duplicate.